src/azure-cli/azure/cli/command_modules/batch/tests/latest/test_batch_data_plane_commands.py

```diff
@@ -219,7 +219,7 @@ def test_batch_jobs_and_tasks(self, resource_group, batch_account_name):
 
         # test create job with missing parameters
         self.kwargs['start'] = datetime.datetime.now().isoformat()
-        with self.assertRaises(SystemExit):
+        with self.assertRaises(Exception):
```

Comment:
Why it's not a AzCLIError ?

Reply:
For test, the main purpose here is to assert a failure. So, I think Exception is enough for these cases.

AzCLIError could work here and act as a fine-grained assertion. While my concern is that a fine-grained binding may bring extra works for further possible adjustments on error types.

Anyway,  I am open for this.